examples/summarize/src/summarize.py

```diff
@@ -0,0 +1,52 @@
+import os
+from airtop import Airtop, SessionConfigV1
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# Setup environment variables
+api_key = os.getenv("AIRTOP_API_KEY")
+if not api_key:
+    print("Error: AIRTOP_API_KEY environment variable must be set.")
+    exit(1)
+
+TARGET_URL = "https://www.nba.com"
+EXTRACT_DATA_PROMPT = "Summarize the content of this page in one paragraph."
+# Prompts used in the TARGET_URL
+
+try:
+    # Initialize AirTop client
+    client = Airtop(api_key=api_key)
+
+    # Create a session configuration
+    configuration = SessionConfigV1(
+        timeout_minutes=10,
+    )
+
+    # Create a session
+    session = client.sessions.create(configuration=configuration)
+    if not session or hasattr(session, "errors") and session.errors:
+        raise Exception(f"Failed to create session: {session.errors}")
+
+    session_id = session.data.id if session.data else None
+    
+    # Create a browser window
+    window = client.windows.create(session_id, url=TARGET_URL)
+    if not window.data:
+        raise Exception("Failed to create window")
+        
+    # Get the window ID
+    window_id = window.data.window_id
+
+    current_content = client.windows.page_query(
+        session.data.id,
+        window_id,
+        prompt=EXTRACT_DATA_PROMPT,
+    )
+    current_result = current_content.data.model_response[:]
+    print(current_result)
+except Exception as e:
+    print(e)
+finally:
+    print("Terminating Airtop session...")
```

Comment:
Let's remove the print statement here since it could be confused with the result:
![image](https://github.com/user-attachments/assets/8c45c8db-c0fd-42cf-874b-9b39b696c184)